packages/zoe/src/contractFacet/exit.js

```diff
@@ -60,7 +61,9 @@ export const makeMakeExiter = baggage => {
         const { state, self } = this;
 
         activeWakers.delete(self);
-        state.zcfSeat.exit();
+        if (!state.zcfSeat.hasExited()) {
```

Comment:
why shouldn't a second exit be an error? (please comment about valid repeated calls)

Reply:
from discussion: it could be exited before the waker wakes. E.g. the contract may have already satisfied the seat and exited it.

please include a comment about that